src/components/dashboard/main-content/course-enrollments/course-cards/CompletedCourseCard.jsx

```diff
@@ -1,36 +1,143 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
 import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { sendTrackEvent } from '@edx/frontend-platform/analytics';
+import { courseEnded } from '../../../../../utils/common';
```

Comment:
tiny nit: add a new line above `import { courseEnded }` so imports from installed packages are grouped together, and local imports are grouped.

also, random thought: perhaps `isCourseEnded` or `hasCourseEnded` to make it more clear its a boolean?